invenio_communities/alembic/de9c14cbb0b2_create_communities_tables.py

```diff
@@ -0,0 +1,220 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Create communities tables."""
+
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+from sqlalchemy.dialects import mysql, postgresql
+
+# revision identifiers, used by Alembic.
+revision = "de9c14cbb0b2"
+down_revision = "90642d415317"
+branch_labels = ()
+depends_on = None
```

Comment:
Since we depend on the `invenio-files-rest` tables, we need a dependency here to the revision where they were introduced